package-res/resources/web/pentaho/data/_doc/IAbstractFilter.jsdoc

```diff
@@ -0,0 +1,85 @@
+/*!
```

Comment:
What is the idea behing creating an interface for each abstract class in a class hierarchy? 
Why not document the class itself?

I think interfaces are neat to describe the structure of plain objects that are used to define how a given class is to be constructed or configured (spec objects).
But I fail to see the point in an hierarchy of classes.